js/components/Breakpoints.js

```diff
@@ -1,24 +1,40 @@
 const React = require("react");
 const dom = React.DOM;
+const { connect } = require("react-redux");
+const { getSources } = require("../queries");
 
-function Breakpoints({ breakpoints }) {
+require("./Breakpoints.css");
+require("../lib/variables.css");
 
-  function onResumeClick() {
-    console.log('click')
-    gThreadClient.resume();
-  }
+const Breakpoints = React.createClass({
```

Comment:
I don't love the class syntax, honestly. You lose auto-bound methods and I don't see much gain. We can still use the `foo() { ... }` syntax for declaring methods in objects, and we probably should start doing that. The only other syntax difference is `class` instead of `createClass`. I forget exactly but I think there were one or two other gotchas with ES6 classes.

(ES6 classes are also not optimized in SpiderMonkey yet, which is a big deal)